compiler/luci-micro/mbed-os/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.9)
```

Comment:
Looks like this file is generated (like other files in this PR :slightly_smiling_face:)
Could you explain how did you get them?

If it is easy to automate, we can do it in CMake scripts, If it is not trivial, you can put this description in readme